src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -69,13 +69,14 @@ const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
     return <BaseSystemChat item={item} index={index} />;
   }
 
-  if (item?.isMe)
+  if (item?.isMe) {
     return (
       <ChatItemMyTextV2
         key={index}
         avatar={handleAvatar()}
         isContinuous={item?.isContinuous}
         message={item?.message}
+        attachments={item?.attachmentJson}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces the `attachments` prop to the `ChatItemMyTextV2` component. Ensure that this component is capable of handling this new prop and that it doesn't break existing functionality. Also, make sure that `item?.attachmentJson` is always in the expected format before passing it as a prop.

```diff
  if (item?.isMe) {
     return (
       <ChatItemMyTextV2
         key={index}
         avatar={handleAvatar()}
         isContinuous={item?.isContinuous}
         message={item?.message}
+        attachments={item?.attachmentJson}
         time={item?.updatedAt}
         username={handleUserName(item)}
         status={item?.status as ChatStatus}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->